packages/vite-plugin-cloudflare/src/node-js-compat.ts

```diff
@@ -1,18 +1,124 @@
-import { createRequire } from "node:module";
+import assert from "node:assert";
+import { cloudflare } from "@cloudflare/unenv-preset";
 import MagicString from "magic-string";
 import { getNodeCompat } from "miniflare";
-import * as unenv from "unenv";
-import type { WorkerConfig } from "./plugin-config";
-import type { Environment } from "vite";
+import { defineEnv } from "unenv";
+import { resolvePluginConfig } from "./plugin-config";
+import type {
+	PluginConfig,
+	ResolvedPluginConfig,
+	WorkerConfig,
+} from "./plugin-config";
+import type { Plugin } from "vite";
+
+/**
+ * A Vite plugin that can provide Node.js compatibility support for Vite Environments that are hosted in Cloudflare Workers.
+ */
+export function nodejsCompatPlugin(pluginConfig: PluginConfig): Plugin {
+	let resolvedPluginConfig: ResolvedPluginConfig;
+	return {
+		name: "vite-plugin-nodejs-compat",
```

Comment:
Let's use this naming convention for child plugins.

```suggestion
		name: "vite-plugin-cloudflare:nodejs-compat",
```

Reply:
d03e2df996b448cb58abbf3ed7c2cfba3a7d7bf1